lib/rules/declaration-block-no-duplicate-properties/__tests__/index.js

```diff
@@ -174,6 +174,95 @@ testRule({
 	],
 });
 
+testRule({
+	ruleName,
+	config: [true],
+	fix: true,
```

Comment:
[suggest] Let's merge new test cases with existing ones like this:

```diff
 testRule({
 	ruleName,
 	config: [true],
+	fix: true,
```

Reply:
> [suggest] Let's merge new test cases with existing ones like this:

You mean adding `fix: true` to the first `testRule` and replacing all rejected tests within it with fixed versions from the second `testRule`?

Reply:
I mean adding `fix: true` to each `testRule`. Then, all rejected test cases should have a `fixed` property, and the diff in test cases should be fewer.